src/main/java/gridss/cmdline/MultipleSamFileCommandLineProgram.java

```diff
@@ -21,19 +22,28 @@
 import picard.cmdline.StandardOptionDefinitions;
 
 import java.io.File;
+import java.io.FileWriter;
+import java.io.BufferedWriter;
 import java.io.IOException;
+import java.nio.file.Files;
 import java.util.List;
 import java.util.Locale;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
+import htsjdk.samtools.*;
+import htsjdk.samtools.reference.FastaSequenceFile;
+import htsjdk.samtools.reference.ReferenceSequence;
 
 public abstract class MultipleSamFileCommandLineProgram extends ReferenceCommandLineProgram {
 	private static final Log log = Log.getInstance(MultipleSamFileCommandLineProgram.class);
-	@Argument(shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, doc="Coordinate-sorted input BAM file.")
+	@Argument(shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, doc="Coordinate-sorted input BAM file.", optional=true)
     public List<File> INPUT;
+
+	@Argument(doc="Supply category instead of empty input file", optional=true)
```

Comment:
```suggestion
	@Argument(doc="Supply list of sample names in the same order as in the assembly output instead of raw reads file to skip generation of candidates from the raw reads file", optional=true)
```